tests/test_interface_ospf.rb

```diff
@@ -21,6 +21,8 @@
 
 # TestInterfaceOspf - Minitest for InterfaceOspf node utility class.
 class TestInterfaceOspf < CiscoTestCase
+  @feature = 'interface_ospf'
```

Comment:
Wondering if it would just be better to skip the indirection of marking the yaml file and mark the minitest directly.  Something like the root :operating_system and :platform keys in the beaker tests (@operating_system, @platform?).  The idea that a minitest has a "primary" feature/yaml file just bothers me, for some reason. Not enough to hold this up, though, since this solution is straightforward enough that we can change our minds later without much penalty.